src/plugins/shared_ux/public/components/redirect_app_links/click_handler.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { ApplicationStart } from 'src/core/public';
+import { getClosestLink, hasActiveModifierKey } from './utils';
+
+interface CreateCrossAppClickHandlerOptions {
+  container: HTMLElement;
+  navigateToUrl: ApplicationStart['navigateToUrl'];
+}
+
+export const createNavigateToUrlClickHandler = ({
+  container,
+  navigateToUrl,
+}: CreateCrossAppClickHandlerOptions): React.MouseEventHandler<HTMLElement> => {
+  return (e) => {
+    if (container === null) {
```

Comment:
`CreateCrossAppClickHandlerOptions` has container as a prop, and it is defined as `HTMLElement` type. This means either:
1. the container is never `null` and this check is redundant
2. the container can be null, so this needs to be specified in interface definition
.
I suspect the latter may be the case. 

Also, in case the double equality here was intentional, rather than accidental,  to be on the safe side, I'd change this if to `!container`, so it covers for `undefined` as well.

Reply:
bacc20b for changes thanks!